docs/install/use-command-line-parameters-to-install-visual-studio.md

```diff
@@ -59,7 +59,7 @@ When you install Visual Studio 2017 RC from a command prompt, you can use the fo
 |  ```--sharedInstallDir <dir>``` | Optional: The target installation directory for shared payloads. |  
 | ```--compatInstallDir <dir>``` | Optional: The target installation directory for legacy compatibility payloads. |  
 |  ```--layoutDir <dir>```, ```--layoutDirectory <dir>``` | Optional: The layout directory in which to find packages.|
-
+|  ```--locale <language-locale>``` | Optional: Change the locale of the user interface for the installer. Setting will be preseved. |
```

Comment:
Not sure you noticed the other change in the suggested sentence. Do you want to say "Change the locale..." or "Change the display language..."?
"display language" may be clearer to users.